### PR TITLE
Added Command Line Tool

### PR DESCRIPTION
**Pull request by briarHarrison**
The command line batch processing tool has been added. I would especially appreciate notes on documentation- if usage is clear, etc.

(Note, I did merge in a few things from the jenkins branch to resolve errors that were discovered as I was working on that. There may be some extra stuff sitting around from then- e.g., `bash_utils`)

### CODE REVIEW

File: usps_service/__main__.py
Comment (jcollard) [inline usps_service/__main__.py]: Could you add linting to the testing process? It will catch this error (expected newline at end of file).

File: usps_service/__main__.py
Comment (jcollard) [inline usps_service/__main__.py:17]: Would it be possible to have the output go to stdout?

File: usps_service/__main__.py
Comment (jcollard) [inline usps_service/__main__.py:67]: Why not print?

**Review by jcollard — COMMENTED** [review]
Some nice work here. I think the tool works well. What was the reasoning for using `sys.stdout.write` over `print`?

For batch processing, it would be very convenient to have the output go to stdout with no other messages (except messages to stderr). This will allow us to pipe together multiple csv processing tools if we'd like to.

Finally, let's wait for the jenkins branch to be merged in before merging this one so we can catch all of the errors.

**Review by jcollard — CHANGES_REQUESTED** [review]
Looks like you accidentally committed some compiled pyc files: `test/__pycache__/test_address_standardization.cpython-36.pyc`
`usps_service/__pycache__/__main__.cpython-36.pyc`

You may want to double check that `__pycache_` directories are in `.gitignore` 

Also, is this file necessary: ` dist/USPSService-0.1.tar.gz`

**Comment by briarHarrison** [comment]
As far as I understand it, the tar is part of what is needed for packaging. It is created when the command to package is run. I'm not sure that external sources would be able to pip install without it? But maybe I am just confused?

**Review by jcollard — APPROVED** [review]
lgmt! Rebase and merge when ready.

**Pull request merged**
